src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Handlers/RetryHandler.cs

```diff
@@ -74,17 +74,22 @@ public class RetryHandler : DelegatingHandler
                 }
                 catch (Exception ex)
                 {
-                    if (ex.IsFatal() || attempt == RetryHandler.MaxAttempts)
+                    if (ex.IsFatal() || attempt == RetryHandler.MaxAttempts - 1)
```

Comment:
How can attempt > RetryHandler.MaxAttempts in for (int attempt = 1; attempt <= RetryHandler.MaxAttempts; ++attempt) ? :)  

---
In reply to: [66539492](https://github.com/Azure/azure-powershell/pull/2412#discussion_r66539492) [](ancestors = 66539492)